src/plugins/unified_search/public/filter_bar/filter_item/filter_item.tsx

```diff
@@ -400,9 +392,35 @@ export function FilterItem(props: FilterItemProps) {
     );
   }
 
+  const renderComponent = () => {
+    switch (renderedComponent) {
+      case 'menu':
+      default:
+        return <EuiContextMenu initialPanelId={0} panels={getPanels()} />;
+      case 'editFilter':
+        return (
+          <EuiContextMenuPanel
+            items={[
+              <div style={{ width: FILTER_EDITOR_WIDTH, maxWidth: '100%' }}>
```

Comment:
nit: is : maxWidth:'100%' required?

Reply:
Thanks for the comment. We can really do without this css property. I will remove it.